upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -488,13 +488,17 @@ func (c *ApplyClusterCmd) Run() error {
 		clusterLifecycle = fi.LifecycleIgnore
 
 	case PhaseSecurity:
+		if c.TargetName == TargetDryRun {
```

Comment:
We shouldn't have to change behaviour based on dryrun vs not - dryrun should hook the process earlier.

Of course if we do need to switch on dryrun, we probably have to fix that

Reply:
Need assistance with this ... I am missing some context.